week-3/reading-list/script.js

```diff
@@ -20,4 +20,26 @@ const books = [
     bookCoverImage: "https://blackwells.co.uk/jacket/l/9780135957059.jpg",
   },
 ];
-
+// document.body.style.width = "100vh";
+const unorderList = document.getElementById("reading-list");
+books.forEach((book) => {
+  const newlist = document.createElement("li");
+  newlist.style.width = "90%";
```

Comment:
To add styles like this, it would be best to make use of the `style.css` file. This makes sure that your styles aren't mixed up with the logic of looping through books